newsroom/wire/filters.py

```diff
@@ -0,0 +1,180 @@
+from typing import Any
+
+from superdesk.core import get_app_config
+from newsroom.search.types import NewshubSearchRequest, BaseSearchRequestArgs, SectionEnum, SearchFilterFunction
+from newsroom.search.config import is_search_field_nested
+from newsroom.search.filters import (
+    get_apply_time_limit_filter,
+    get_apply_filters,
+    get_apply_highlights,
+    apply_query_string,
+    apply_ids_filter,
+    apply_date_range,
+    apply_advanced_search,
+    prefill_user,
+    prefill_company,
+    prefill_products,
+    prefill_args_from_topic,
+    apply_section_filter,
+    apply_company_filter,
+    apply_products_filter,
+    validate_request,
+)
+from newsroom.search.utils import query_string_for_section
+from newsroom.settings import get_setting
+
+from .types import DateRangeQuery, TimeFilter
+
+
+class WireSearchRequestArgs(BaseSearchRequestArgs):
+    """Search arguments for wire items"""
+
+    #: If ``True``, will allow searching previous versions of an article
+    ignoreLatest: bool = False
+
+    #: If ``True``, will exclude embargoed items from this search
+    exclude_embargoed: bool = False
+
+    #: If ``True``, will only include embargoed items in this search
+    embargoed_only: bool = False
+
+    #: If ``True``, will apply the configured NewsOnly filter for this search
+    newsOnly: bool = False
```

Comment:
just out of curiosity, why is the camelCase in this one?

Reply:
So we don't have to change the front-end code. I have changed this now, so we can support camelCase from front-end and use snake_case on the back-end